srcpkgs/nodejs-lts-10/patches/ppc32.patch

```diff
@@ -15,7 +15,7 @@
        'msvs_disabled_warnings!': [4244],
  
        'conditions': [
-+        [ 'host_arch=="mips" or host_arch=="mipsel" or host_arch=="ppc"', {
++        [ 'target_arch=="mips" or target_arch=="mipsel" or target_arch=="ppc" or target_arch=="arm"', {
```

Comment:
Given that this now covers arm, I think you should change the file name.

Reply:
I'll just split that part into its own patch because the other bits of the file are definitely `ppc32`-specific.